Add percentPosition Masonry option to available attributes

Vyygir opened the pull request:
I needed the `percentPosition` option from Masonry v4 available in a build I'm doing, so I've updated the `attributesMap` object so it's accepted.

Let me know if something's not quite right!

shershen08 commented:
Thanks for the input @Vyygir 
Did you check and have a demo that this parameter properly works with masonry ?

Vyygir commented:
I updated the version I'm using in a project first to test the functionality, then forked the repo, updated the code, and made a PR. So no demo from the outset but I'll implement one for you soon

shershen08 commented:
fixed in `0.10.16`
https://github.com/shershen08/vue-masonry/commit/b8804d01907729b7be3e97a9aeae79e80c1692ba

Thanks for your input!
